qa/src/test/resources/credentials.properties

```diff
@@ -0,0 +1,22 @@
+## ---------------------------------------------------------------------------
```

Comment:
Shure. As the conten is copied from Active MQ.
Do you think if I add Kapua copyright and keep content as it is, that will be ok. 
Because content is as it is.